tpm2tools/keys.go

```diff
@@ -133,3 +135,160 @@ func (k *Key) PublicKey() crypto.PublicKey {
 func (k *Key) Close() {
 	tpm2.FlushContext(k.rw, k.handle)
 }
+
+// Seal seals the sensitive byte buffer to the provided PCRs under the owner
+// hierarchy using the SHA256 versions of the provided PCRs.
+// The Key k is used as the parent key.
+func (k *Key) Seal(rw io.ReadWriter, pcrs []int, sensitive []byte) ([]byte, []byte, error) {
```

Comment:
It might be better to have a struct that contains both parts of the "Wrapped Key", namely it it the public template and the encrypted secret. I think it's needed to make it clear to users that `priv` is not a secret.

Reply:
Done.